creme/proto/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Prototype- based methods."""
```

Comment:
nit: no need for whitespace after the dash.

More important: could you possible add a short paragraph to give some intuition on what are prototype-based methods? I don't think that a lot of people are aware of them.

Reply:
Fixed the typo and added a short description.